mycodo/controllers/controller_trigger.py

```diff
@@ -243,13 +236,7 @@ def initialize_variables(self):
                     pwm_duty_cycle, ended = self.get_method_output(
                         self.trigger.unique_id_1)
                     if not ended:
-                        output_channel = db_retrieve_table_daemon(OutputChannel).filter(
-                            OutputChannel.unique_id == self.trigger.unique_id_3).first()
-                        if output_channel:
-                            self.set_output_duty_cycle(
-                                self.trigger.unique_id_2,
-                                pwm_duty_cycle,
-                                output_channel=output_channel.channel)
+                        self.set_output_duty_cycle(pwm_duty_cycle)
```

Comment:
the db access was repeated multiple times and therefore moved into the set_output_duty_cycle method.